DDG4/plugins/Geant4FastPhysics.cpp

```diff
@@ -0,0 +1,107 @@
+//==========================================================================
```

Comment:
Was this file really supposed to be included in this MR ? I do not see the relationship